communities/asu-phoenix/community-assessment.md

```diff
@@ -0,0 +1,121 @@
+
+- 20000+ students enrolled at Arizona State University 
+    - 5,650 enrolled in Computing Program
+- 200-300 at large scale hackathons    
+
+- No target number, but want to ensure that all students, at such a large scale university, have a chance to learn more about technology and be a part of the community.
+
+- Recently, more hackathon groups have been arising and events that are more hands on have been able to see more than hundreds of students participaring! 
+    - The change is our school has become more accepting wih their new drive to be more innovative and have more of an entreprenial mindset. 
+
+### Resources
+- What is currently available to people in your community to learn about tech?
+    - Workshops run by SoDA
+    - Coding Competitions
+    - Career Fairs
+    - Company Recruitment Panels
+
+#### Exisiting organizations and Tech Influences
+- ASU, Large University, Multiple Organizations
+    - Computer Science Organizations 
+        - Mobile Devs
+        - Video Game Developer Club
+        - Software Developer Association 
+        - Women in Computer Science 
+        - Technology + Innovation Lab 
+        - Sun Devil Robotics
+        - Intern Supply 
+        - PWN Devils 
+        - CodePath
+        - Desert Hacks
+    - Digital Culture Organizations 
+        - Digital Culture Underground 
+        - [Other](https://artsmediaengineering.asu.edu/resources/students/clubs-organizations) 
+    - Engineering Organizations 
+        - Fulton Ambassadors 
+        - Fulton Student Council 
+        - Polytechnic Campus Orgs
+        - [Fulton Student Orgs](https://fso.engineering.asu.edu/student-org-directory/)
+
+- Phoenix Area, many partnerships + meetups 
+    - Galvanize, great co-working/event space
+    - #YesPHX Movement (Tech Startup Movement)
+    - Meetups 
+        - Code For America
+        - VR/AR Hangouts 
+        - IoT Meetups 
+    - Hackerrnest socials 
+    - Local Companies 
+        - GoDaddy 
+        - Intel 
+        - Amazon
+        - American Express 
+
+### Accessibility
+- There is no central area where all the resources from each club is placed
+    - Most are put on SunDevilSync or a link provided during the event
+- Club Meetings usually happen Monday-Thursday in between 6-8pm
+
+### Interests
+
+- Mostly, but we would like to welcome anyone interested in technology
+- There are several other majors that have a tech influence that we would love to get involved
+    - Digital Culture (Art)
+    - Technical Management (Polytechnic Campus)
+    - Graphic Information Systems (Polytechnic Campus)
+    - Computer Systems Engineering
+    - Computing Informatics Systems (Business)
+
+### Goals
+
+- Most people get involved to make connections, find jobs/internships, learn new skills
+- We would like to provide a larger scale event to really get everyone who has these goals all in one. Furthermore, have them develop connections with the existing communties and encourage them to stay invovled. 
```

Comment:
This is great. With so many groups existing, being a bridge between them is a super valuable service to provide, and providing a large event that others can unify around is a great way of doing that.